src/Microsoft.ML.Core/Utilities/ResourceManagerUtils.cs

```diff
@@ -328,4 +336,15 @@ public static ResourceDownloadResults GetErrorMessage(out string errorMessage, p
         private static extern int chmod(string pathname, int mode);
 #pragma warning restore IDE1006
     }
+
+    public class WebClientResponse : WebClient
```

Comment:
This class doesn't seem to be providing a whole lot of functionality. Is this strictly necessary? Is it possible to read the response directly from WebClient?

Reply:
You resolved this comment. And removed the reference to WebClientResponse from the function above. Do you still need this class?

---
In reply to: [394102879](https://github.com/dotnet/machinelearning/pull/4943#discussion_r394102879) [](ancestors = 394102879)

Reply:
Apologies for the confusion, I marked comments as resolved as I fixed them locally but before I pushed my changes to keep track of what I worked on.